proto/shentu/oracle/v1alpha1/oracle.proto

```diff
@@ -105,12 +105,12 @@ message CoinsProto {
     repeated cosmos.base.v1beta1.Coin coins = 1 [ (gogoproto.moretags) = "yaml:\"amount\"", (gogoproto.nullable) = false, (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins" ];
 }
 
-message TxTask {
+message AtxTask {
```

Comment:
Should we rename the structure name or only the fields name?

Reply:
The confuse of tx_hash has been resolved by Bin's PR. 
This is to resolve the confuse of application chain transaction and shentu chain transaction.

Reply:
All changes are in this commit, I closed my other PR.